scripts/ci/utils/upload-sourcemaps

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
@nLight anything else we have to do to enable it on 1.11 branch? Or shall we extract it from this PR for now?

Reply:
It should be good. Basically Sentry only cares about the release and source maps. If they are there it's good to go